projects/step-frontend/src/lib/angularjs/js/controllers/plans.js

```diff
@@ -30,7 +30,8 @@ angular
       'st-table',
       '/partials/plans/planSelectionTable.html',
       null,
-      'glyphicon glyphicon-file'
+      'glyphicon glyphicon-file',
+      'description'
```

Comment:
Should we keep this legacy registration?
For migrated tables, entities are registered in ng2+ modules

Reply:
You are right, we can remove the ones that are migrated already, I will also remove the old icon since it is not used anywhere anymore.